README.md

```diff
@@ -166,3 +166,9 @@ style="some-style" >}}
 ## Getting help
 
 If you run into an issue that isn't answered by this documentation or the [`exampleSite`](https://github.com/zwbetz-gh/minimal-bootstrap-hugo-theme/tree/master/exampleSite), then visit the [Hugo forum](https://discourse.gohugo.io/). The folks there are helpful and friendly. **Before** asking your question, be sure to read the [requesting help guidelines](https://discourse.gohugo.io/t/requesting-help/9132). Feel free to tag me in your question, my forum username is [@zwbetz](https://discourse.gohugo.io/u/zwbetz/summary).
+
+## Stackbit Deploy
```

Comment:
Update the table of contents at the top by adding:
```
- [Stackbit Deploy](#stackbit-deploy)
```

Reply:
Updated in the commit c2ca5ee